tests/e2e/helpers/mobileSearch.js

```diff
@@ -29,10 +29,11 @@ export function runMobileSearchTests(br) {
 
       //testing successful search
       await t
-        .selectText(nav.searchBox.child('.BRsearchInput'))
+        .selectText(nav.searchBox.find('[name="query"]'))
```

Comment:
perhaps we still need this to be a bit more specific? do both inputs have that name?

Reply:
They do, however the local `nav` is `br.nav.mobile`, so it's scoped to the mobile search query input.

Reply:
ah ok, is that why the desktop search input didn't have the search value that was passed in on page load?